content/departments/product-engineering/engineering/code-graph/code-intelligence/glossary.md

```diff
@@ -0,0 +1,374 @@
+# Code Intel Glossary
+
+### build system
+
+A tool that is used to transform code into a format where it can be executed, usually by carrying out a series of build steps (~almost always organized as a [directed acylic graph](#directed-acylic-graph)), each of which invokes a more lower-level tool (such as a [compiler](#compiler)).
+
+A build system understands the relations between different files, [modules][#module] and [packages][#package].
+
+In the context of Code Intel, for out-of-the-box support, an [indexer](#indexer) needs to invoke the build system in the right way. (TODO: Clarify this!)
+
+Examples:
+
+- Bazel, Buck, Pants, Meson: Support many languages.
+- Maven, Gradle, Ant: Primarily used for Java, Kotlin and Scala.
+- CMake: Primarily used for C and C++, but also used for other projects. Strictly speaking, CMake is a "meta build system"; it generates build instructions which are executed by another build system (such as Make or Ninja).
+
+### Code Intel
```

Comment:
I think we can keep this section because we do abbreviate Code Intel internally. I would probably clarify this is an internal abbreviation and that we don't recommend using it outside of Sourcegraph to prevent confusion

Reply:
Is it because of the renaming effort or something else? Having a shorthand seems useful when talking to external folks compared to the more verbose "code intelligence."